uusi.cabal

```diff
@@ -34,7 +34,7 @@ source-repository head
 common common-options
   build-depends:
     , base   >=4.8 && <5
-    , Cabal  ^>=3.2 || ^>=3.4 || ^>=3.10
+    , Cabal  ^>=3.2 || ^>=3.4 || ==3.6.3 || >=3.8
```

Comment:
Can we use `^>= 3.8 || ^= 3.10` instead of `>= 3.8` here?

Reply:
Then we'll see us again when GHC 9.8 releases.
If there is an issue, it can be reported, just like I did with the other PR. But it's really annoying when the build fails just because of a version restriction. And I'd rather avoid the cyclic dependency of building uusi with uusi

Reply:
@berberman I've updated the PR to include your suggested change. I don't really agree with limiting the upper bound, but that discussion has surely been had already.